### ISSUE TITLE
Newly created indices use long for time field

### ISSUE THREAD

**Issue by daradermody** [regel/loudml#74]
The type of the time field is long instead of date. The following is the mapping for the datasink:
```
{
    "ml-output-2018.11.08": {
        "mappings": {
            "doc": {
                "properties": {
                    "@timestamp": {
                        "type": "long"
                    },
                    "is_anomaly": {
                        "type": "boolean"
                    },
                    "lower_value": {
                        "type": "float"
                    },
                    "model": {
                        "type": "text",
                        "fields": {
                            "keyword": {
                                "type": "keyword",
                                "ignore_above": 256
                            }
                        }
                    },
                    "score": {
                        "type": "float"
                    },
                    "upper_value": {
                        "type": "float"
                    },
                    "value": {
                        "type": "float"
                    }
                }
            }
        }
    }
}
```
As a result, there are issues with some date histogram visualisations. Can the index mappings be updated to ensure the time field is of type `date` before indexing results?

**Comment by regel** [comment]
Hi @daradermody, 
Agree. timestamp mapping type must be updated, and BTW "model" mapping type should be a keyword.

**Comment by regel** [comment]
fixed in 1.5.0
